front/public/locales/fr/infraEditor.json

```diff
@@ -139,25 +139,25 @@
     "title": "Direction"
   },
   "DirectionalTrackRange": {
-    "description": "Un intervalle sur un TIV est un intervalle avec une direction associée.",
+    "description": "Un intervalle sur une section de voie est un intervalle avec une direction associée.",
     "properties": {
       "begin": {
-        "description": "Distance de début par rapport au début du TIV (en mètres).",
+        "description": "Distance de début par rapport au début de la section de voie (en mètres).",
         "title": "Début"
       },
       "direction": {
         "description": "Description de la direction."
       },
       "end": {
-        "description": " Distance de fin par rapport au début du TIV (en mètres).",
+        "description": " Distance de fin par rapport au début de la section de voie (en mètres).",
```

Comment:
```suggestion
        "description": "Distance de fin par rapport au début de la section de voie (en mètres).",
```

Reply:
Is this leading space a trick to workaround a missing CSS padding somewhere? Or is it inconsequential?

Reply:
if possible we should avoid such workarounds, to make the work of translators in other languages easier,
is it a subject for a new issue?

Reply:
Translating all the app should be a new issue I think